onnxruntime/core/framework/tensor_allocator_with_mem_pattern.h

```diff
@@ -21,7 +21,8 @@ class TensorAllocatorWithMemPattern : public ITensorAllocator {
   bool is_sealed_ = false;
   const ExecutionPlanBase& seq_plan_;
 
-  common::Status AllocatePlannedBuffers() {
+  common::Status AllocatePlannedBuffers(size_t& planned_memory_size_in_byte) {
```

Comment:
>AllocatePlannedBuffers [](start = 17, length = 22)

Since now this method also fetches the planned memory size, maybe try re-name it to AllocatePlannedBuffersAndReturnSize?

Reply:
Ok.

---
In reply to: [440499802](https://github.com/microsoft/onnxruntime/pull/4241#discussion_r440499802) [](ancestors = 440499802)